lib/cli.rb

```diff
@@ -27,9 +27,19 @@ def run
       end
     end
 
+    command :logout do |c|
+      c.syntax = 'ccli logout'
+      c.description = 'ccli logout'
```

Comment:
Maybe put a more descriptive description in here.
Or atleast make it differ from the syntax. Something along the lines of `Logs out of the ccli`